web/src/beta/lib/reearth-ui/components/Breadcrumb/index.tsx

```diff
@@ -16,17 +16,21 @@ export type BreadcrumbProp = {
   separator?: ReactNode;
   size?: "normal" | "large";
   onClick?: (id?: string) => void;
+  ariaLabel?: string;
+  dataTestid?: string;
 };
 
 export const Breadcrumb: FC<BreadcrumbProp> = ({
   items = [],
   separator = " / ",
   size,
-  onClick
+  onClick,
+  ariaLabel = "breadcrumb",
+  dataTestid = "breadcrumb"
```

Comment:
Although giving the default value seems okay when component used once and user didn't override the dataTestid, but If you render multiple `Breadcrumb` component on the same page and don’t override data-testid, then, Queries like getByTestId("breadcrumb") will fail with: Found multiple elements with the same data-testid. to avoid these issue we might ignore default value